src/commands/Hello.ts

```diff
@@ -0,0 +1,29 @@
+import { SlashCommandBuilder } from "discord.js";
```

Comment:
I think casing should be consistent.
So this should also be camelCased.

Reply:
> I think casing should be consistent. So this should also be camelCased.

What makes you think this is inconsistent? The only way we could achieve it being camelCased would be something like the below (which isn't consistent):
`import { SlashCommandBuilder as slashCommandBuilder } from "discord.js";`

Reply:
I meant the file name sorry.